src/auth/webhook-secret.guard.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Injectable,
+  UnauthorizedException,
+} from '@nestjs/common';
+import * as crypto from 'crypto';
+import { Request } from 'express';
+import { Env } from 'src/env';
+
+@Injectable()
+export class WebhookSecretGuard implements CanActivate {
```

Comment:
PullProofAI

Ensure consistent naming conventions. Consider renaming `WebhookSecretGuard` to follow a consistent naming pattern.
```diff
- export class WebhookSecretGuard implements CanActivate {
+ export class WebhookSignatureGuard implements CanActivate {
```

### Example response

No further issues found.


<!-- This is an auto-generated comment by PullProofAI -->